standards-catalogue/Rakefile

```
@@ -58,9 +58,19 @@ namespace :schema do
       end
       raise unless errors.all?
     end
+
+    desc "Validate licences are schema compliant"
+    task :licences do
+      schema = schema("licence")
+      organisations = YAML.load_file("data/licences.yml")
+      errors = organisations.map do |f|
+        is_valid_schema(schema, f[1])
```

I'm happy to raise this as a follow-up, so we can tweak the existing organisations code, or if you'd like you can do both here, so they'd be i.e.:

```suggestion
      errors = organisations.map do |_, organisation|
        is_valid_schema(schema, organisation)
```

---

Added another commit to address these https://github.com/alphagov/data-standards-authority/pull/126/commits/cc03ee1ff5fab704516818ed7bfec6060085bc79